README.md

```diff
@@ -5,7 +5,7 @@ Extract Binaries from Apple's Dyld Shared Cache to be useful in a disassembler.
 # Installation
 
 ```
-python3 -m pip install dyldextractor
+pip3 install git+https://github.com/donato-fiore/DyldExtractor.git@iOS-18-Support
```

Comment:
I don't think this change should be included in the PR

Reply:
I agree.

Reply:
Additionally, there are a lot of files that I don't think you intended to include, like `.DS_Store` and the `build` folder.

Reply:
I reverted the installation command back to the original and added `.DS_Store` and `build/` to the ignore list